archive_viewer/mixins/traces_table.py

```diff
@@ -199,8 +198,9 @@ def __init__(self, parent: QObject) -> None:
             # PV currently does nothing, this is a remnant
             # From when we would have the pv_list open in a new window
             if button_text == "PV":
-                # TODO: Either give a function for this button or replace it
-                pass
+                self.PVButton = button
+                self.PVButton.setCheckable(True)
+                self.PVButton.clicked.connect(self.showPVList)
```

Comment:
The PVButton is unchecked by default, so the PV list is hidden by default. Is this intended?
If so, I think it makes more sense to show it by default.
```suggestion
                self.PVButton = button
                self.PVButton.setCheckable(True)
                self.PVButton.setChecked(True)
                self.PVButton.clicked.connect(self.showPVList)
```

Reply:
That was intentional, and sounds good I'll change it